cfg.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2014-2019 Yubico AB - See COPYING
+ */
+
+#ifndef CFG_H
+#define CFG_H
+
+#include <stdio.h>
+
+#define CFG_DEFAULT_PATH (SYSCONFDIR "/security/pam_u2f.conf")
+#define CFG_MAX_FILE_SIZE 1024 // Arbitrary
```

Comment:
This looks like a little on the low side to me. :slightly_smiling_face: 

Does the maximum file size need to be documented somewhere?

Reply:
Yeah, maybe. let's say 4K? (or 1.44 MiB, if you want to fill up a floppy)

I will document it.